### PR TITLE
contrib/libftdi1: Fix pkgspec

### PR DESCRIPTION
**Pull request by indrora**
Fix description: this is libftdi not yajl.

### CODE REVIEW

**Comment by nekopsykose** [comment]
the commit should be titled `contrib/libftdi1: fix pkgdesc` or similar

File: contrib/libftdi1/template.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 hostmakedepends = ["cmake", "ninja", "pkgconf"]
 makedepends = ["libconfuse-devel", "libusb-devel"]
 checkdepends = ["boost-devel"]
-pkgdesc = "Yet Another JSON Library"
+pkgdesc = "Library for communicating with FTDI USB devices"
```
Comment (nekopsykose) [inline contrib/libftdi1/template.py:9]: bump the pkgrel too since it affects the package metadata

**Comment by indrora** [comment]
I realize these commits should be squashed together in an ideal world. When I get access to a real machine (not a phone) I'll do the squash.

**Comment by nekopsykose** [comment]
saved you a few seconds :)

**Comment by indrora** [comment]
Awesome. Thank you.

**Pull request merged**